Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -527,19 +530,21 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
             i++;
         }
 
-        prevYPosForDraw = yPos;
+        if ([runs objectAtIndex:0] && static_cast<_CTRun*>([runs objectAtIndex:0])->_dwriteGlyphRun.glyphCount != 0) {
```

Comment:
Yea this was just a workaround from hitting this issue, this way will be stomped by the better fix during merge